MapboxNavigation/FeedbackViewController.swift

```diff
@@ -3,29 +3,25 @@ import MapboxCoreNavigation
 import AVFoundation
 
 extension FeedbackViewController: UIViewControllerTransitioningDelegate {
-    func animationController(forDismissed dismissed: UIViewController) -> UIViewControllerAnimatedTransitioning? {
+    public func animationController(forDismissed dismissed: UIViewController) -> UIViewControllerAnimatedTransitioning? {
```

Comment:
If we make these methods public, we’ll need to give them less awkward names (`forDismissing`?) and documentation and consider making them accessible to Objective-C.

Reply:
Noting, we have no control over the naming conventions of this method since it's a func defined on `UIViewControllerTransitioningDelegate `.

Reply:
So it is! 😄 (By the way, the Objective-C method name looks fine; it’s just the Swift name that’s funky.) Oh well, carry on.